Makefile

```diff
@@ -240,6 +240,10 @@ deploy-olm: bundle bundle-build bundle-push ## Builds and pushes the operator OL
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	make deploy-yaml | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
+.PHONY: undeploy-olm
+undeploy-olm: ## Undeploys the operator from the cluster (used only if operator was installed via OLM)
```

Comment:
You probably want to add `operator-sdk` as a dep here

Reply:
Thanks! done